snapdtool/cmdutil.go

```diff
@@ -112,9 +112,32 @@ func CommandFromSystemSnap(name string, cmdArgs ...string) (*exec.Cmd, error) {
 	cmdPath := filepath.Join(root, name)
 
 	if from == "snapd" {
-		// We do not need to specify the linker or library
-		// path from files coming from the snapd snap.
-		return exec.Command(cmdPath, cmdArgs...), nil
+		// check if the elf interpreter invoked by the binary will work
+		// without any tweaks, which is true if the snps are mounted at
+		// /snap, or do we need to invoke it directly,
```

Comment:
"or do we need to invoke it directly," <- sounds like a question, unwanted ',' at the end